controller/model_controller.go

```diff
@@ -3,19 +3,30 @@ package controller
 import (
 	"dummy/repository"
 	"html/template"
+	"log"
 	"net/http"
 
 	"github.com/go-chi/chi/v5"
 )
 
 func ListModel(w http.ResponseWriter, r *http.Request) {
-	models := repository.GetModels()
+	models, err := repository.GetModels()
+
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
1) Only log, but not returning any response?
2) If `err != nil`, then I assume that `models` is nil. If `models == nil`, then you will cause [panic](https://gobyexample.com/panic) when generating the template.